pkg/awx/connection.go

```diff
@@ -263,7 +305,7 @@ func (c *Connection) rawGet(path string, query url.Values) (output []byte, err e
 	if err != nil {
 		return
 	}
-	if response.StatusCode != 200 {
+	if response.StatusCode > 202 {
```

Comment:
@jhernand this is needed since after creating the new token the server returns the `created` 201 status code.